packages/react/src/components/Dialog/Dialog.tsx

```diff
@@ -211,15 +212,15 @@ class Dialog extends AutoControlledComponent<ReactProps<DialogProps>, DialogStat
             },
             overrideProps: {
               content: (
-                <>
+                <Flex gap="gap.smaller" hAlign="end">
```

Comment:
@kuzhelov this means introducing an extra DOM node, any suggestions on how to avoid this?
I can only think about adding a `marginRight` variable and override it for `Button` when needed

also @layershifter as code owner of this component

Reply:
I think that here it is absolutely fine to introduce additional DOM node - as, if you would imagine what one would do to achieve the same effect with plain HTML, it would still be about introducing container DOM node

Reply:
cool, thanks for looking into it